packages/graphql-mapping-template/src/ast.ts

```diff
@@ -360,6 +380,10 @@ export function block(name: string, exprs: Expression[]): CompoundExpressionNode
   return compoundExpression([comment(`[Start] ${name}`), ...exprs, comment(`[End] ${name}`)]);
 }
 
+export function methodCall(methodName: ReferenceNode, ...params: Expression[]): CompoundExpressionNode {
+  return compoundExpression([methodName, parens(compoundExpression(params, ', '))], '', !RECURSE_INDENT);
```

Comment:
`!RECURSE_INDENT` might not be needed here as a default argument is already specified on the compoundExpression itself.

Reply:
The default is true though (for backwards compatibility with the old behavior when that was not a parameter) and here it needs to be false